src/app/core/ui/navigation/menu-item/menu-item.component.scss

```diff
@@ -14,10 +14,57 @@
   padding-left: 20px;
 }
 
+.submenu-item-container {
+  display: flex;
+  align-items: center; // Vertically align icons and content
+  flex-direction: row; // Horizontal layout
+  gap: 10px; // Space between drag handle, item, and remove icon
+  padding: 8px;
+  margin: auto;
+}
+
 .indent-item {
   margin-left: 8px;
   padding-left: 8px;
   width: auto;
   border-end-end-radius: 0;
   border-start-end-radius: 0;
 }
+
+.cdk-drag-preview {
```

Comment:
do we need all these classes here? I think now they are only needed in the admin components, not in this one?

Can you check and remove it here?